tests/sentry/tasks/test_statistical_detectors.py

```diff
@@ -126,32 +103,12 @@ def test_run_detection_options(
         "statistical_detectors.enable": enable,
     }
 
-    features = {
-        "organizations:performance-statistical-detectors-ema": [project.organization.slug]
-        if performance_project
-        else [],
-        "organizations:profiling-statistical-detectors-ema": [project.organization.slug]
-        if profiling_project
-        else [],
-    }
-
-    if performance_project_option_enabled:
-        ProjectOption.objects.set_value(
-            project=project,
-            key="sentry:performance_issue_settings",
-            value={
-                "transaction_duration_regression_detection_enabled": performance_project_option_enabled
-            },
-        )
-
-    with freeze_time(timestamp), override_options(options), Feature(features):
+    with freeze_time(timestamp), override_options(options):  # , Feature(features):
```

Comment:
```suggestion
    with freeze_time(timestamp), override_options(options):
```
Did you mean to keep this comment?